api/models/resource.util.js

```diff
@@ -206,13 +206,13 @@ exports.setOrganizations = async (resource, orgs) => {
 };
 
 exports.setUser = async (resource, user) => {
-  return await queries.execUpdateSetManyToMany(
+  return await queries.execUpdateSetManyToOne(
+    User,
+    null,
+    user,
     Resource,
     'resources',
-    resource,
-    User,
-    'user',
-    [user]
+    [resource]
   );
 };
```

Comment:
I'm thinking this should actually be moved to userUtil. As like setResources.

Reply:
Oh :o that makes sense